skll/config.py

```diff
@@ -276,7 +276,7 @@ def _parse_config_file(config_path):
 
     # produce warnings if hasher_features is set but feature_hasher
     # is not set correctly
-    if hasher_features > 0:
+    if not feature_hasher and hasher_features > 0:
```

Comment:
Good catch, but this could be simplified to:

```py
elif hasher_features > 0:
```

because we're checking if `feature_hasher` is set right about this.  I imagine that was the original intention for this, but there was a typo.